src/Frost/Effects/Github.hs

```diff
@@ -0,0 +1,39 @@
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Frost.Effects.Github where
+
+import Polysemy
+import Polysemy.Error
+import FrostError
+import qualified GitHub.Endpoints.Issues as GHI
+import qualified GitHub.Data.Options as GHO
+import qualified GitHub.Data.Issues as GHDI
+import qualified GitHub.Internal.Prelude as GIP
+import PolysemyContrib
+import qualified Data.Vector as DV
+import qualified Data.List.Split as DLS
+
+type Issue = String
+
+data Github m a where
+  Issues :: String -> Github m [Issue]
+
+makeSem ''Github
+
+runGithubPure :: [Issue] -> Sem (Github ': r) a -> Sem r a
+runGithubPure issues = interpret $ \case
+  Issues _ -> return issues
+
+runGithubIO :: (Member (Embed IO) r, Member (Error FrostError) r) => Sem (Github ': r) a  -> Sem r a
+runGithubIO = interpret $ \case
+  Issues repo -> do
+    (username, reponame) <- either throw (\b -> return b) (parseRepo repo)
+    fromEitherSem $ embed $ issuesForFrost username reponame
+  where
+    issuesForFrost username reponame = do
+      mis <- GHI.issuesForRepo username reponame GHO.optionsAnyMilestone
+      return $ either (\e -> Left $ FrostError $ show e) (\l -> Right $ fmap (\i -> show $ GHDI.issueTitle i) (DV.toList l) ) mis
```

Comment:
`(\e -> Left $ FrostError $ show e)` can be just function composition `Left . FrostError . show`

Reply:
`(\l -> Right $ fmap (\i -> show $ GHDI.issueTitle i) (DV.toList l) ) ` I think [not sure] can be just
`Right . (fmap (show . GHDI.issueTitle ) . DV.toList`